### ISSUE TITLE
Development halted? :(

### ISSUE THREAD

**Issue by Vastrix** [fragmer/PASaveEditor#9]

**Comment by mstefarov** [comment]
Yeah, I haven't worked on this for a while -- not enough free time. I'll probably go back and update this project whenever PA 1.0 comes out.

**Comment by itsnotvalid** [comment]
It comes out now.

**Comment by Cschwalb** [comment]
Looks like I will take a peek at this and throw some PR's if the owner likes it :)
